tests/test_weekday_calendar.py

```diff
@@ -1,14 +1,22 @@
+from __future__ import annotations
 from unittest import TestCase
+from collections import abc
 
+import pytest
 import pandas as pd
 import pandas.testing as tm
 from pytz import UTC
 
 from exchange_calendars.weekday_calendar import WeekdayCalendar
-
-from .test_exchange_calendar import ExchangeCalendarTestBase
+from exchange_calendars import ExchangeCalendar
+from .test_exchange_calendar import (
+    ExchangeCalendarTestBase,
+    ExchangeCalendarTestBaseProposal,
+    Answers,
+)
 
 
+@pytest.mark.skip
```

Comment:
Is this purposely skipped?

Reply:
Yes. I included a note on this at the end of the first [comment](https://github.com/gerrymanoim/exchange_calendars/pull/71#issue-700763978).  

Basically, introducing the side option has allowed the 24 hour calendars to be truly 24 hour - the open times have been moved 'back' one minute to represent the actual open time. These calendars can only take side as "left" or "right". They default to "right" to maintain the original behaviour; each minute remains associated with the same session as perviously. Whilst the behvaiour hasn't changed (save for a change to the open time) some of the old tests fail as they were designed on the assumption that the open minute and close minute would both be a trading minutes - this assumption continues to hold for all the other calendars (which default to side "both") but not the 24 hour ones.

All the 24 hour calendars are subjected to and pass the new test suite.